docs/Reference/Engine-API-Objects.md

```diff
@@ -0,0 +1,69 @@
+| `logsBloom`     | *Data*, 256 Bytes     | Bloom filter for light clients to quickly retrieve related logs. |
+| `prevRandao`    | *Data*, 32 Bytes      | Difficulty for this block.                                       |
+| `blockNumber`   | *Quantity*, 64 Bits   | Block number of block containing this transaction.               |
+| `gasLimit`      | *Quantity*, 64 Bits   | Maximum gas allowed in this block.                               |
+| `gasUsed`       | *Quantity*, 64 Bits   | Total gas used by all transactions in this block.                |
+| `timestamp`     | *Quantity*, 64 Bits   | Unix timestamp for block assembly.                               |
+| `extraData`     | *Data*, 0 to 32 Bytes | Extra data field for this block.                                 |
+| `baseFeePerGas` | *Quantity*, 256 Bits  | The block's [base fee per gas](../Concepts/Transactions/Transaction-Types.md#eip1559-transactions). This field is empty for blocks created before [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md). |
+| `blockHash`     | *Data*, 32 Bytes      | Hash of the execution block.                                     |
+| `transactions`  | *Array*               | Array of transaction objects, each object is a list representing `TransactionType`, `TransactionPayload`, or `LegacyTransaction` as defined in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718). |
+
+## Fork choice state object
+
+Parameter for [`engine_forkchoiceUpdatedV1`](Engine-API-Methods.md#engine_forkchoiceupdatedv1).
+
+| Key | Type | Value |
+|-----|:----:|-------|
+| `headBlockHash`      | *Data*, 32 Bytes | Block hash of the head of the canonical chain. |
+| `safeBlockHash`      | *Data*, 32 Bytes | "Safe" block hash of the canonical chain under certain synchrony and honesty assumptions. This value MUST be either equal to or an ancestor of `headBlockHash`. |
+| `finalizedBlockHash` | *Data*, 32 Bytes | Block hash of the most recent finalized block. |
+
+## Payload attributes object
+
+Parameter for [`engine_forkchoiceUpdatedV1`](Engine-API-Methods.md#engine_forkchoiceupdatedv1).
+
+| Key | Type | Value |
+|-----|:----:|-------|
+| `timestamp`             | *Quantity*, 64 Bits | Value for the `timestamp` field of the new payload.              |
+| `prevRandao`            | *Data*, 32 Bytes    | Value for the `prevRandao` field of the new payload.             |
+| `suggestedFeeRecipient` | *Data*, 20 Bytes     | Suggested value for the `feeRecipient` field of the new payload.|
+
+## Payload status object
+
+Returned by [`engine_newPayloadV1`](Engine-API-Methods.md#engine_newpayloadv1) and [`engine_forkchoiceUpdatedV1`](Engine-API-Methods.md#engine_forkchoiceupdatedv1).
+
+| Key | Type | Value |
+|-----|:----:|-------|
+| `status`          | *enum*           | Either `"VALID"`, `"INVALID"`, `"SYNCING"`, `"ACCEPTED"`, `"INVALID_BLOCK_HASH"`, or `"INVALID_TERMINAL_BLOCK"`. |
```

Comment:
See my comment about the `accept.txt` file change.

Reply:
Fixed